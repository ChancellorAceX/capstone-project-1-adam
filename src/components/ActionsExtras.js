import React from 'react';

export default function ActionExtras(props) {
  return (
    <React.Fragment>
      <tr>
        <th>{props.h}</th>
      </tr>
      <tr>
        <td>{props.p}</td>
      </tr>
    </React.Fragment>
  );
}