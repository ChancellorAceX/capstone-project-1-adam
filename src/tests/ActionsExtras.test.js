import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ActionsExtras from '../components/ActionsExtras';

describe('ActionsExtras component', () => {
  const score=['Strength', 'str', 'strmod']
  const monster={str:17,strmod:3}
  it('renders the complete table unit', () => {
    const wrapper = shallow(<ActionsExtras 
      h={'Heated Body'}
      p={'A creature that touches the azer or hits it with a melee attack while within 5 feet of it takes (1d10) fire damage.'}
    />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});