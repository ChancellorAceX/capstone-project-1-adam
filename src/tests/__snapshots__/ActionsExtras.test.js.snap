// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionsExtras component renders the complete table unit 1`] = `
<Fragment>
  <tr>
    <th>
      Heated Body
    </th>
  </tr>
  <tr>
    <td>
      A creature that touches the azer or hits it with a melee attack while within 5 feet of it takes (1d10) fire damage.
    </td>
  </tr>
</Fragment>
`;
